assets/scss/components/citadel/forms/_forms.scss

```diff
@@ -209,6 +209,10 @@
         @include breakpoint("large") {
             width: auto;
         }
+
+        &::placeholder {
+            color: $formInput-placeholder-color;
+        }
```

Comment:
As I can see with [can i use](https://caniuse.com/?search=%3A%3Aplaceholder) ::placeholder won't be working in ie. Should we do something with that?

Reply:
done